.travis.yml

```diff
@@ -1,9 +1,8 @@
-
 # Stuff shared by all builds
 install:
   - python setup.py install
 script:
-  - python -m pytest
+  - :
```

Comment:
It does nothing. `python -m pytest` raises an error because there are no tests currently, and we aren't ready to run automated linting, so right now I think it should just make sure that the installation works.

Reply:
ah I see. I actually disagree with doing this. It means that the first person coming to write tests (if that person isn't you) needs to know to come back and add `python -m pytest`.

I think it would be better to actually put in `python -m pytest` and then just make one test that has `assert True == True` and create one or more "good first issue" issues describing tests you want people to add one day. That way, more junior developers who want to help could focus on writing tests and not have to know about Travis!

Reply:
Good idea